lib/shared/units.js

```diff
@@ -70,7 +44,7 @@ exports.bytesToMegabytes = (bytes) => {
  * > '7.8 GB'
  */
 exports.bytesToClosestUnit = (bytes) => {
-  if (_.isNumber(bytes)) {
+  if (_.isNumber(bytes) && _.isFinite(bytes)) {
```

Comment:
Same here, no need to check whether it's a number, as isFinite() checks that

Reply:
Weirdly enough I thought to check if `_.isFinite` could replace the null-check but not this obvious case...